### ISSUE TITLE
Tagging for stable releases [bitnami/milvus]

### ISSUE THREAD

**Issue by roszkowski** [bitnami/containers#65678]
### Name and Version

[bitnami/milvus1:2.3.13]

### What is the problem this feature will solve?

Hello! 

I've been checking multiple images and once I actually use them in my helm chart and deploy - all seem to have "-dev" suffix once I log to them into attu. It raises questions that I couldn't answer myself.

What is the latest non-dev version of bitnami/milvus?

### What is the feature you are proposing to solve the problem?

I would propose adding appropriate status to release tag names like "-dev" or "-beta" to avoid unnecessary checks.

**Comment by carrodher** [comment]
That shouldn't be the case since we don't provide `dev`, `alpha`, `beta`, etc code. In the case of Milvus and Attu, we fetch the source code from the upstream repos, at this moment the latest source code was obtained from https://github.com/milvus-io/milvus#refs/tags/v2.4.0 and https://github.com/zilliztech/attu#refs/tags/v2.3.10 respectively.

How are you obtaining that `dev` suffix?

**Comment by roszkowski** [comment]
@carrodher thanks for reply. I get it in attu once I log in there:
<img width="732" alt="image" src="https://github.com/bitnami/containers/assets/22658241/8b832015-025a-429b-bb74-79e35dbf43d7">

and this is what I provide in values.yaml:
```
milvus:
  image:
    tag: 2.3.11
```

**Comment by carrodher** [comment]
Can you see the same without setting a specific tag? I mean, removing the `image.tag` block from the values.yaml

**Comment by roszkowski** [comment]
@carrodher yes it was the case as well, that's why I started checking other versions

**Comment by andresbono** [comment]
Hi I can reproduce the issue. For instance, with the Bitnami chart milvus:7.0.0:

```console
$ helm install milvus oci://registry-1.docker.io/bitnamicharts/milvus --version 7.0.0 -f .vib/milvus/runtime-parameters.yaml
```

![image](https://github.com/bitnami/containers/assets/1057973/5dabcdaa-fbe9-43b3-b4ae-07db0f3b934e)

The version can also be observed in the logs:

```console
$ kubectl logs deploy/milvus-proxy milvus | grep '^Version:'
Version:   v2.3.12-dev
```

The issue should be fixed with the next release of milvus 2.4.1 (release still in progress):

```
$ milvus run proxy
...
Version:   v2.4.1
```

**Comment by dgomezleon** [comment]
The new version 2.4.1 does not contain the dev tag. Container has been released and chart should be out in a few minutes.
